velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -240,11 +244,23 @@ std::string toCallSql(const core::CallTypedExprPtr& call) {
         sql << signature->nameOf(i);
       }
 
-      sql << ") -> " << toCallSql(body);
+      sql << ") -> " << toCallSql(body, {}, {});
     } else {
       VELOX_NYI();
     }
   }
+
+  if (!sortingKeys.empty()) {
+    VELOX_CHECK_EQ(sortingKeys.size(), sortingOrder.size());
```

Comment:
Should we move this check to the beginning of this function as their size should always be the same?